source/agora/consensus/validation/Block.d

```diff
@@ -55,6 +60,8 @@ version (unittest)
     it as a spent output. See the `findNonSpent` function in the
     unittest for an example.
 
+    As a special case, the genesis block follows its own validation logic.
```

Comment:
This comment is outdated. It should say genesis is rejected in this function, and only accepted in the other function that was introduced.